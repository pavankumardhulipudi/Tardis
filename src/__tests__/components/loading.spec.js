import 'react-native';
import React from 'react';
import Loading from '../../components/loading';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('testing export components', () => {
  const tree = renderer.create(<Loading/>).toJSON();
  expect(tree).toMatchSnapshot();
});
