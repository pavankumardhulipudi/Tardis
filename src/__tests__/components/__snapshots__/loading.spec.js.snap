// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing export components 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
